NN_2L2H2O_poly2d/python.out

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
python.out is not machine readable easily,

I suggest to do instead 2 .csv files, `single_precision_keras_prediction.csv` and `double_precision_keras_prediction.csv`